app/frontend/components/common/navbar/navbarUnauth.js

```diff
@@ -10,24 +9,27 @@ const NavbarUnauth = connect(
   actions
 )(({openGenerateMnemonicDialog}) =>
   h(
```

Comment:
I was thinking more like this usage of `h1`.
```js
 h(
    'nav',
    {class: 'navbar'},
    h(
      'div',
      {class: 'navbar-wrapper'},
      h(
        'h1',
        {class: 'navbar-heading'},
        h('span', {class: 'navbar-title'}, 'AdaLite - Cardano Wallet'),
        h(
          'a',
          {class: 'navbar-logo', href: '/'},
          h('img', {
            src: 'assets/adalite-logo.svg',
            alt: 'AdaLite - Cardano Wallet',
          })
        )
      ),

```

and the styles for `.navbar-title`

```css
.navbar-title {
  width: 0;
  height: 0;
  text-indent: -999999px;
  display: inline-block;
}
```

Reply:
Great :)